update-nft-metadata-in-treasury/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "update-nft-metadata-in-treasury",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "type": "module",
+  "dependencies": {
+    "@hashgraph/sdk": "^2.49.0",
+    "dotenv": "^16.3.1"
```

prefer exact match on dependency numbers
(optional here, since this a demo snippet)
```suggestion
    "@hashgraph/sdk": "2.49.0",
    "dotenv": "16.3.1"
```